scripts/release-fdroid.sh

```diff
@@ -0,0 +1,78 @@
+version_name=$(cat VERSION)
```

Comment:
Missing shebang. Also the filename clashes with the `make release-fdroid` target. Maybe `make-fdroid-pr.sh`?